js/iiif-iabookreader_strawberry.js

```diff
@@ -32,25 +35,19 @@
                   bookId: node_uuid,
                   enableSearch: true,
                   searchInsideUrl: '/do/' + node_uuid + '/flavorsearch/all/ocr/',
+                  plugins: {
+                    textSelection: {
+                      enabled: has_ocr,
+                      singlePageDjvuXmlUrl: '/do/' + node_uuid + '/flavorsearch/all/ocr/djvuxml/{{pageIndex}}',
+                    },
+                  },
                   padding: 11,
                 };
                 var br = new BookReader(options);
                 br.init();
-                // Check if Book has or not OCR using Ajax callback
```

Comment:
@giancarlobi this needs to come back. We are using Ajax to identify if there is OCR

Reply:
@DiegoPino The issue is that we need to know if ocr exists also to enable/disable text selection some lines above, for that reason I changed to this code, if not, we have to make a sync call before init to enable/disable text selection or any other solution you consider ok